CoreScriptsRoot/Modules/VR/LaserPointer.lua

```diff
@@ -416,17 +416,17 @@ do --Input handlers - can we eventually move this stuff to PlayerScripts? CoreSc
 
 	function LaserPointer:setTriggerActionEnabled(enabled)
 		if enabled then
-			ContextActionService:BindCoreAction("TeleportTriggerImpl", function(...) self:onTeleportTriggerAction(...) end, false, self.inputUserCFrame == Enum.UserCFrame.RightHand and Enum.KeyCode.ButtonA or Enum.KeyCode.ButtonX)
+			ContextActionService:BindAction("TeleportTriggerImpl", function(...) self:onTeleportTriggerAction(...) end, false, self.inputUserCFrame == Enum.UserCFrame.RightHand and Enum.KeyCode.ButtonA or Enum.KeyCode.ButtonX)
```

Comment:
The justification for using BindAction here, in my mind, is that this code should really be under ControlScript anyway, but we can't have it there because ParabolaAdornment is limited to CoreScripts right now. I'm making a note to move this module if ParabolaAdornment gets that limitation lifted.